installer/conf/omsagent.d/vmware_esxi.conf

```diff
@@ -0,0 +1,24 @@
+<source>
+  type syslog
+  format /^(?<EventTime>\S+)\s{0,4}(?<HostName>\S+)\s{0,4}(?<ProcessName>\S+)\:\s{0,4}(?<SyslogMessage>.*)$/
+  port 1514
```

Comment:
https://github.com/Microsoft/OMS-Agent-for-Linux/blob/keikodoc/docs/VMware-Instruction.md

customer will set it up on ESXi Host.